cpp/src/graphar/filesystem.cc

```diff
@@ -233,8 +233,13 @@ Status FileSystem::WriteTableToFile(const std::shared_ptr<arrow::Table>& table,
     break;
   }
   case FileType::PARQUET: {
+    auto schema = table->schema();
+    auto column_num = schema->num_fields();
     parquet::WriterProperties::Builder builder;
     builder.compression(arrow::Compression::type::ZSTD);  // enable compression
+    for (int i = 0; i < column_num; ++i) {
```

Comment:
It appears that RLE encoding is currently applied to all columns. Could we adjust this so that RLE is used only on the label columns?

Reply:
for sure!